__tests__/app.test.js

```diff
@@ -112,9 +112,9 @@ describe('PATCH  /api/articles/:article_id', () => {
 })
 
 describe('GET /api/articles', () => { 
-    test('return an articles array of article objects with custom columns and sorted by date in descending order', () => {
+    test('return an articles array of article objects with custom columns', () => {
```

Comment:
The original description seemed better for this. Not sure what custom columns means?